scripts/post-promote-tasks-utils.ts

```diff
@@ -77,3 +84,29 @@ export async function getBaseAmpVersion(
     }
   }
 }
+
+export async function getPullRequestDetails(
+  pullNumber: number
+): Promise<PullRequestDetails | void> {
+  const octokit = new Octokit();
+  const {data: pr} = await octokit.rest.pulls.get({
+    owner: 'ampproject',
+    repo: 'cdn-configuration',
+    pull_number: pullNumber,
+  });
+
+  if (!pr.merge_commit_sha || !pr.merged_at) return;
```

Comment:
Personally I prefer throwing an error in this situation, and catching it in the caller, to simplify the return type (~~`| void`~~) - but that's a personal choice